k2/csrc/fsa.h

```diff
@@ -10,6 +10,7 @@
 #include <cstdint>
 #include <tuple>
 #include <vector>
+#include <utility>
```

Comment:
Have you installed `clang-format`?

It should reorder the headers for you automatically.

Reply:
let me check.
```
Package              Version
-------------------- ----------
absl-py              0.9.0
argcomplete          1.9.4
astor                0.6.2
bleach               1.5.0
cachetools           4.0.0
certifi              2019.11.28
chardet              3.0.4
clang-format         9.0.0

```

Reply:
seems it will not reorder in my machine...strange.

Reply:
Have you configured https://github.com/google/vim-codefmt ?

Reply:
Yes, I have done it before. Anyway, changed the order manually for now.